ieeeuoft/newhacks-site-2022
# NH-35: Added links to dashboard

## Overview

- Resolves #NH-35
- Added links to dashboard page
![image](https://user-images.githubusercontent.com/43456930/199628116-3adee9c8-c3bd-4a1a-8a98-1f6b2c2537fa.png)



## Unit Tests Created

- n/a


## Steps to QA

- Sign in as participant account and verify links are correct and accurate on the Dashboard Page
